cypress/src/support/commands.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable jest/no-standalone-expect */
+/// <reference types="cypress" />
+import 'os';
+
+Cypress.Commands.add('loginAsGuest', () => {
+  window.localStorage.setItem('@backstage/core:SignInPage:provider', 'guest');
```

Comment:
Does this still work? I'm pretty sure that we have to visit the page first to get the window object like before?